### DISCUSSION TITLE
Most effective way to search across multiple collections?

### DISCUSSION THREAD

**Discussion by vrdn-23** [qdrant/qdrant-client#857] [Q&A]
Is there an effective way to search across multiple collections with the same query? I can see APIs for batch search and hybrid search, but I'm interested in running the same query across multiple collections. Is there any API method that currently lets us do that? 

Similar to what [OpenSearch aliases](https://opensearch.org/docs/latest/api-reference/index-apis/alias/) allow to set a single alias for multiple collections.

**Comment by joein** [comment]
Hey @vrdn-23, sorry for the late response, there is no convenience API to search over multiple collection with the same query, in order to do so, you'd need to do multiple requests on the client's side
